src/Bookings.js

```diff
@@ -1,18 +1,46 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
+import SearchResults from "./SearchResults.js";
 // import FakeBookings from "./data/fakeBookings.json";
+import CustomerProfile from "./CustomerProfile.js";
 
 const Bookings = () => {
+  // task -14
+  const [bookings, setBookings] = useState([]);
+
+  // task 16
+  useEffect(() => {
+    // console.log("loading for the first time")
+    fetch(`https://cyf-react.glitch.me`)
+      .then(res => res.json())
+      .then(data => setBookings(data));
+  }, []);
+
   const search = searchVal => {
-    console.info("TO DO!", searchVal);
+    // console.info("TO DO!", searchVal);
+    const newBookings = bookings.filter(booking => {
+      return (
+        booking.firstName.toLowerCase() === searchVal.toLowerCase() ||
+        booking.surname.toLowerCase() === searchVal.toLowerCase()
+      );
+    });
+
+    setBookings(newBookings);
   };
```

Comment:
Using the filter method here works really well, good job using `.toLowerCase() ` too to check for both upper and lower case matches :)
To improve this search function, think about how you might make it work if you were to make multiple searches for people - for example, at the moment if you search for 'john' and then search for 'judi' afterwards it doesn't bring up judi's profile (because the filtered bookings from when you searched for John are now saved in state.) Can you think of a way around this?